src/js/options.ts

```diff
@@ -79,7 +80,13 @@ export interface ReactNativeOptions extends BrowserOptions {
    *
    * @default true
    * */
-   enableOutOfMemoryTracking?: boolean;
+  enableOutOfMemoryTracking?: boolean;
+
+  /**
+   * Set data to the inital scope
+   * @deprecated Use `Sentry.configureScope(...)`
+   */
+  initialScope?: CaptureContext;
```

Comment:
`initialScope` is inherited from JS options

Reply:
@AbhiPrasad or @jennmueng do you know if there's a better way to deprecate a feature that comes from the JS SDK and is available to the RN SDK, but we don't want to expose it to the RN SDK, so we rather than override the attribute and annotate it with @deprecated is that fine? thanks

Reply:
I think annotating with `@deprecated` is the best we got, do we also want to filter out the option before it gets passed down?

Reply:
no, not really, it still works, so just deprecated is fine for now, thanks @AbhiPrasad